test/lit.cfg

```diff
@@ -311,6 +311,40 @@ if test_sdk_overlay_dir is not None:
 test_options = os.environ.get('SWIFT_TEST_OPTIONS')
 config.swift_test_options = ' ' + test_options if test_options else ''
 
+def get_clang_library_dir():
+    llvm_lib_clang = os.path.join(config.llvm_libs_dir, 'clang')
+    if not os.path.exists(llvm_lib_clang):
+        return ''
+
+    versions = os.listdir(llvm_lib_clang)
+    if not len(versions):
+        return ''
+
+    version = max(versions)
+    return os.path.join(llvm_lib_clang, version, 'lib',
+                        platform.system().lower())
+
+config.clang_library_dir = get_clang_library_dir()
```

Comment:
Even without compiler-rt around, (1) llvm-macosx-$target/lib/ contains libclangBasic.a and friends (so it seems like libclang_rt belongs in the llvm build directory too?), and (2) lib/swift/clang doesn't get created.

I'm assuming creating a symlink from `swift-macosx-$target/lib/swift/clang` to `llvm-macosx-$target/lib/clang/$version/lib` isn't a permanent solution.

Do you know which cmake file or build option is responsible for creating lib/swift/clang? And do you have any pointers for testing with Xcode?